Formula/svt-hevc.rb

```diff
@@ -0,0 +1,21 @@
+class SvtHevc < Formula
+  desc "Scalable Video Technology for HEVC Encoder"
+  homepage "https://01.org/svt"
+  url "https://github.com/OpenVisualCloud/SVT-HEVC/archive/v1.4.3.tar.gz"
+  sha256 "08bf2b1075609788194bf983d693ab38ed30214ec966afa67895209fd0bf2179"
+  head "https://github.com/OpenVisualCloud/SVT-HEVC.git"
+
+  depends_on "cmake" => :build
+  depends_on "yasm" => :build
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/SvtHevcEncApp --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
As I said above this formula requires some sample video input. I'm unsure if there is some video files inside default macOS install. Or we can download some sample video, e.g. Big Buck Bunny trailer?

Reply:
I had found this `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/Frameworks/ReplayKit.framework/srecorder.mp4`

Is it fine to do depend test on xcode, run xcode-select and grab this file?

Reply:
Oh, it requires yuv input. I may just dynamically download [this](https://media.xiph.org/vqeg/TestSeqences/Reference/src1_ref__625.yuv) public domain test sequence?

Reply:
I think it might be best to download something public using a resource block instead of depending on Xcode.